synapse/models/infotech.py

```diff
@@ -429,6 +432,9 @@ def getModelDefs(self):
                 ('it:dev:repo:issue', ('guid', {}), {
                     'doc': 'An issue raised in a repository.',
                 }),
+                ('it:dev:repo:issue:label', ('guid', {}), {
```

Comment:
Lets discuss. I think label probably doesn't need the guid abstraction and could likely be just a `:labels` array on the issue node :+1: